src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -97,12 +100,15 @@ public static class TextCatalog
         }
 
         /// <summary>
-        /// Normalizes incoming text in <paramref name="inputColumnName"/> by changing case, removing diacritical marks, punctuation marks and/or numbers
-        /// and outputs new text as <paramref name="outputColumnName"/>.
+        /// Creates a <see cref="TextNormalizingEstimator"/>, which normalizes incoming text in <paramref name="inputColumnName"/> by changing case,
+        /// removing diacritical marks, punctuation marks and/or numbers and outputs new text as <paramref name="outputColumnName"/>.
```

Comment:
it reads as it's doing casing removing diacritics and punctuations, and only for numbers you have option to decide.